### DISCUSSION TITLE
New projects are organization-level only?

### DISCUSSION THREAD

**Discussion by borekb** [community/community#5489] [Projects and Issues] [answered]
Will new projects be enabled for a specific repository as well? Currently, they are organization-only which I'm not sure if it's a temporary limitation or a permanent decision.

**Comment by Katsute** [comment, marked as answer]
At the bottom of https://github.com/features/issues (roadmap)

> 1. Ability to create new projects (beta) at both user and repo levels

**Reply by borekb** [reply]
Thanks, I missed this. (BTW, issues roadmap probably isn't tracked in https://github.com/github/roadmap, is it?)
